src/Microsoft.AspNet.Http.Extensions/StreamCopyOperation.cs

```diff
@@ -10,47 +11,58 @@
 namespace Microsoft.AspNet.Http
```

Comment:
Change the namspace to Microsoft.AspNet.Extensions. This will not be a commonly used type.